chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -510,29 +510,35 @@ def raise_argument_errors(self):
             raise ValueError(
                 "MVE Weighting calibration can only be carried out with MVE or Evidential loss function models."
             )
+        if self.num_models == 1:
```

Comment:
Should this be `!=` instead of `==`?

Reply:
No, this is specifically a method for finding the weights to do a weighted average of an ensemble of models. If you only have one model, it has the whole 100% and you can't calibrate that.